app/pandas.ipynb

```diff
@@ -234,18 +584,101 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 1,
-   "outputs": [],
    "source": [
-    "# Write your code here"
+    "df[\"Absolute difference\"] = df[\"Age\"] - df[\"Average age\"]\n",
```

Comment:
The calculation of the absolute difference should use the `abs()` function to ensure the result is non-negative. Use `df['Absolute difference'] = abs(df['Age'] - df['Average age'])`.